content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -59,6 +59,10 @@ An example of such file exists as a project-specific override in the AutomatedTe
 It is recommended to add your `editorpreferences.setreg` file to the `<project-path>/user/Registry` directory as a user-specific override. Files in the user directory are ignored by git and won't be tracked for changes.
 {{< /note >}}
 
+{{< note >}}
+In the latest version, prefab override management in Entity Outliner is enabled by default.
```

Comment:
probably good to indicate the release number here like 23.05 to avoid confusion on what 'latest' means here. Or say something like 'starting in 23.05 release,.....'